airflow/contrib/operators/awsbatch_operator.py

```diff
@@ -94,11 +100,21 @@ def execute(self, context):
         )
 
         try:
-            response = self.client.submit_job(
-                jobName=self.job_name,
-                jobQueue=self.job_queue,
-                jobDefinition=self.job_definition,
-                containerOverrides=self.overrides)
+            if self.parameters is None:
```

Comment:
Do we need the `if` construction here? Can't we just pass None parameters? Or do `parameters =self.parameters or {}`

Reply:
The API does not accept None parameters or {}, both triggers KeyError exception.